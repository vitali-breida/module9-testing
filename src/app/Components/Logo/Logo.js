import PropTypes from "prop-types";
export default function Logo(props) {
  return (
    <div align={props.align}>
      <b>netflix</b>roulette
    </div>
  );
}

Logo.defaultProps = {
  align: "left"
};

Logo.propTypes = {
  align: PropTypes.oneOf(["left", "center"])
};
