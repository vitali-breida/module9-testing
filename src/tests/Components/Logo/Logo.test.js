import Logo from "../../../app/Components/Logo/Logo";
import { render } from "@testing-library/react";

it("renders correctly", () => {
  const { asFragment } = render(<Logo align="center" />);
  expect(asFragment()).toMatchSnapshot();
});
