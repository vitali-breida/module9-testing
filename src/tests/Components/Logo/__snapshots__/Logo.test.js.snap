// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    align="center"
  >
    <b>
      netflix
    </b>
    roulette
  </div>
</DocumentFragment>
`;
